apps/pie-microsite/src/utilities/helpers.js

```diff
@@ -27,12 +31,22 @@ const capitaliseFirstLetter = string => {
     return capitalised;
 };
 
+/**
+ * Checks whether string can convert to number
+ * @param {string} string
```

Comment:
think this needs to accept a string OR number, based on your unit tests :)

Reply:
Or I would rename to something like isStringANumber or something and remove the unit test you added. It's a little confusing otherwise.

Reply:
ah yep, will change the param to string or number, thank you! 😄